components/VotingPower/utils.js

```diff
@@ -0,0 +1,38 @@
+import { ethers } from 'ethers';
+/**
+ *
+ * @param {string} balanceInWei
+ * @returns formatted balance with appropriate suffix
+ */
+export const formatWeiBalance = (balanceInWei) => {
+  const balanceInEther = ethers.utils.formatEther(balanceInWei);
+
+  const formatNumberWithSuffix = (number) => {
+    if (number >= 1e9) {
+      return `${(number / 1e9).toFixed(1)}B`;
```

Comment:
while this is a nice approach, i feel off about the "rounding up" effect that Number.toFixed(1) has.

ideally we would round down, so the user is not confused as to why they cannot delegate in a scenario where they don't have enough voting power